test/e2e/network/service.go

```diff
@@ -1768,7 +1768,7 @@ var _ = common.SIGDescribe("Services", func() {
 		t.Name = "slow-terminating-unready-pod"
 		t.Image = imageutils.GetE2EImage(imageutils.Agnhost)
 		port := int32(80)
-		terminateSeconds := int64(100)
+		//terminateSeconds := int64(100)
```

Comment:
we need to keep this, the test need to define the container used in the deployment

Reply:
what would be the point of just waiting 100s to stop the container? (I shouldn't have left the commented line there though)

Reply:
is what the test is testing `should create endpoints for unready pods"` the pod gets unready during that period.

Is not recommended to do drive-by changes in the same commit, per example, in this case removing this line completely broke the test